libs/openFrameworks/utils/ofFileUtils.cpp

```diff
@@ -31,7 +31,12 @@ namespace{
         try{
             return std::filesystem::canonical(ofFilePath::join(ofFilePath::getCurrentExeDir(),  "../../../data/")).string();
         }catch(...){
-            return ofFilePath::join(ofFilePath::getCurrentExeDir(),  "../../../data/");
+			try{
+			return std::filesystem::canonical(ofFilePath::join(ofFilePath::getCurrentExeDir(),  "../Resources/data/")).string();
+			}catch(...){
+			
+				return ofFilePath::join(ofFilePath::getCurrentExeDir(),  "../../../data/");
+			}
         }
```

Comment:
I would love to get this in. Essentially auto detect if there is a data/ folder in resources and use that, otherwise use the traditional data folder.  

Have you tested this though to see if the try catch correctly works if no `data` folder is found in `Resources/`

Reply:
You could also use the `std::filsystem::canonical` overload that takes a `std::error_code` to avoid exceptions here entirely.